builds/volto-latest.yml

```diff
@@ -0,0 +1,86 @@
+steps:
+
+  # "kaniko" is a build runner whose sole job is to build
+  # container images optimised for use on k8s.
+  # destination is just where on our infrastructure we store
+  # these images.
+  # by default it looks for Dockerfile in the repo root and builds
+  # that though other locations can be specified.
+  # For why we build the image twice with different tags, see
+  # the cicd README.
+  - name: 'gcr.io/kaniko-project/executor:latest'
+    args:
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:latest
+      - >-
+        --destination=${_IMAGE_REPOSITORY}/volto-test:$SHORT_SHA
+      - context=dir://volto
+      - Dockerfile=Dockerfile-test
+
+  # Builds Volto-test latest container from the artifact registry and Volto tests are run using container volume
```

Comment:
in hindsight this is a misleading comment, can you make it "Run volto tests using the volto test container (see above step""